client/my-sites/types/post-type-unsupported/index.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import includes from 'lodash/includes';
+
+/**
+ * Internal dependencies
+ */
+import localize from 'lib/mixins/i18n/localize';
+import EmptyContent from 'components/empty-content';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	isJetpackSite,
+	isCurrentUserCapableForSite,
+	getSiteSlug
+} from 'state/sites/selectors';
+
+/**
+ * Constants
+ */
+const CONFIGURABLE_TYPES = [ 'jetpack-portfolio', 'jetpack-testimonial' ];
+
+function PostTypeUnsupported( { translate, canManage, siteSlug, type } ) {
+	const isConfigurableType = includes( CONFIGURABLE_TYPES, type );
+	let title, line, action, actionUrl;
+
+	if ( isConfigurableType && canManage ) {
+		switch ( type ) {
+			case 'jetpack-portfolio':
+				title = translate( 'Portfolios are not enabled' );
+				break;
+
+			case 'jetpack-testimonial':
+				title = translate( 'Testimonials are not enabled' );
+				break;
+		}
+
+		line = translate( 'Activate custom content types in your site settings' );
+		action = translate( 'Manage Settings' );
+		actionUrl = '/settings/writing/' + siteSlug;
+	} else {
+		title = translate( 'Content type unsupported' );
+		line = translate( 'Your site does not support this content type' );
+	}
+
+	return <EmptyContent { ...{ title, line, action, actionURL: actionUrl } } />;
+}
+
+PostTypeUnsupported.propTypes = {
+	translate: PropTypes.func,
+	jetpack: PropTypes.bool,
+	canManage: PropTypes.bool,
+	siteSlug: PropTypes.string,
+	adminUrl: PropTypes.string,
```

Comment:
>Is this prop being used?

I should really know better by now to let propTypes slip past you 😄  It was used in an earlier iteration where I'd expected to need to redirect Jetpack users to their wp-admin dashboard, before realizing that we can activate the module directly within Calypso site settings screen. I'll remove it.